### PR TITLE
Bug 1883614: Warn user when restarting BMH

### PR DESCRIPTION
**Pull request by rawagner**
Screenshot from 2021-01-08 09-34-06

@andybraren are you ok with the message ? (its the same as non-graceful poweroff)

### CODE REVIEW

File: frontend/packages/metal3-plugin/src/components/modals/RestartHostModal.tsx [hunk 1]
```diff
@@ -41,19 +39,12 @@ const RestartHostModal = ({
     [host, close, handlePromise],
   );
 
-  const text = nodeName
-    ? t(
-        'metal3-plugin~The bare metal host {{name}} will be restarted gracefully after all managed workloads are moved.',
```
Comment (andybraren) [inline frontend/packages/metal3-plugin/src/components/modals/RestartHostModal.tsx:46]: WDYT of adding back a tiny bit of `<p>` text above the warning that reads:

> The host will be powered off and on again.

**Review by andybraren — APPROVED** [review]
One small thing, but otherwise lgtm!

File: frontend/packages/metal3-plugin/src/components/modals/RestartHostModal.tsx (hunk 1 above)
Comment (andybraren) [reply frontend/packages/metal3-plugin/src/components/modals/RestartHostModal.tsx:46]: I know it's a bit obvious 🙂 but I think we typically have _something_ in the body of modals explaining what's going to happen before warnings/alerts.

Comment (rawagner) [reply frontend/packages/metal3-plugin/src/components/modals/RestartHostModal.tsx:46]: sure, added!

**Review by jtomasek — APPROVED** [review]

**Review by jtomasek — APPROVED** [review]
/lgtm

**Pull request merged**